txp/Pos/Txp/Toil/Logic.hs

```diff
@@ -178,28 +176,20 @@ verifyGState curEpoch txAux txFee = do
 verifyBootEra
     :: forall ctx m .
        ( MonadError ToilVerFailure m
-       , HasLens' ctx GenesisStakeholders
+       , HasLens' ctx GenesisWStakeholders
        , MonadReader ctx m)
     => EpochIndex -> EpochIndex -> TxAux -> m ()
 verifyBootEra curEpoch unlockEpoch txAux = do
-    let bootEra = isBootstrapEra curEpoch unlockEpoch
-    bootHolders <- views (lensOf' @GenesisStakeholders) unGenesisStakeholders
+    let bootEra = isBootstrapEra unlockEpoch curEpoch
```

Comment:
That's what I mean: https://github.com/input-output-hk/cardano-sl/pull/1298/files